### DISCUSSION TITLE
将编辑器内嵌使用

### DISCUSSION THREAD

**Discussion by linonetwo** [OpenWebGAL/WebGAL#556] [Q&A] [answered]
我最近在做独立游戏想直接用 WebGAL 做对话部分，其它部分是还有我自己写的战斗部分什么的。在提供 MOD 编辑器时，我可以把我自己的战斗部分的编辑器内嵌进游戏里，但是 WebGAL 的编辑器我也能这么做吗？还是说只能给出链接引导 MOD 开发者下载额外的安装包安装 WebGAL 编辑器呢？

内嵌的一个好处是不用额外安装，另一个好处是锁版本了比较不容易冲突。想请教一下这个目前能做吗，允许做吗？

**Comment by MakinoharaShoko** [comment, marked as answer]
只要遵循 MPL-2.0 就可以。这个协议总体来说比较宽松，只要求对修改的部分开源，而不是整个项目。
